test/test_torch.py

```diff
@@ -3314,24 +3314,6 @@ def test_randn(self):
         torch.randn(SIZE, SIZE, out=res2)
         self.assertEqual(res1, res2)
 
-    def test_slice(self):
```

Comment:
This reduces test coverage. Would it make sense to simply leave the test, but convert  to index notation?

Reply:
yes, that makes a lot of sense to do